docs/docs-content/architecture/networking-ports.md

```diff
@@ -110,60 +128,60 @@ You can expose inbound port 22 for SSH if you would like to access your cluster
 :::
 
 
+## Self-Hosted Network Communications and Ports
 
-</TabItem>
 
-<TabItem label="NATS" value="nats">
+<Tabs groupId="architecture">
+<TabItem label="gRPC" value="gRPC">
 
-## SaaS Network Communications and Ports
 
-The following ports must be reachable from a network perspective for Palette SaaS to function correctly.
+![On-prem network diagram](/architecture_networking-ports_on_prem_network-diagram-grpc.png "#title="network diagram")
 
+</TabItem>
 
-![SaaS Network Diagram with ports](/architecture_networking-ports_network-diagram_nats.png "title=SaaS Network Diagram with ports")
+<TabItem label="NATS" value="nats">
 
+![On-prem network diagram](/architecture_networking-ports_on_prem_network-diagram-nats.png "#title="network diagram")
 
 
-#### SaaS Managed
+</TabItem>
+</Tabs>
 
+The following ports must be reachable from a network perspective for Palette self-hosted to function correctly.
 
-![SaaS network diagram displaying the network paths for edge](/architecture_networking-ports_saas-network-diagram-edge_nats.png)
 
+<Tabs groupId="architecture">
+<TabItem label="gRPC" value="gRPC">
 
-The following ports must be reachable from a network perspective for Palette to operate properly.
 
-## Management Platform
+### Management Platform
 
 |Port            |Direction|Purpose                   |    
 |:---------------|:---------|:-----------------------|
-|HTTPS (tcp/443) |INBOUND        |Browser/API access to management platform|
-|NATS (tcp/4222) |INBOUND        |Agent running inside connecting to management platform|
+|HTTPS (tcp/443) |INBOUND        |Browser/API access to management platform, gRPC|
+|NATS (tcp/4222) |INBOUND        |Message Bus for workload clusters [Deprecated]|
+|HTTPS (tcp/443) |OUTBOUND       |vSphere vCenter API,  Registry (packs, integrations), Pack containers, app updates, gRPC|
+|HTTPS (tcp/6443)|OUTBOUND       |Workload K8s cluster API Server|
 
 
-## Workload Cluster
+### Workload Cluster
 
 
-|Port            |Direction | Purpose|
+|Port |Direction | Purpose|
 |:---------------|:---------|:--------------|
 |HTTPS (tcp/443) |OUTBOUND | API access to management platform|
-|NATS (tcp/4222) |OUTBOUND       |Registry (packs, integrations), Pack containers, Application Updates|
-|NATS (tcp/4222) |OUTBOUND       |Registry (packs, integrations), Pack containers, Application Updates|
-
-:::info
+|NATS (tcp/4222) |OUTBOUND       |Agent communication via message bus. [Deprecated] |
```

Comment:
```suggestion
|NATS (tcp/4222) |OUTBOUND       |Agent communication via message bus [Deprecated] |
```

I noticed that the other sentences don't have a final dot. Could we remove them to be consistent?